components/bl-rating-component/styles/index.less

```diff
@@ -0,0 +1,51 @@
+// you can use LESS variables from the current Theme and app extensions of the UI-Container
+@bl-customComponent-rating-themeColor: @themePrimary;
+@bl-customComponent-rating-backgroundColor: @appBackgroundColor;
+@bl-customComponent-rating-textColor: @appTextColor;
+
+@bl-customComponent-rating-iconWidth: 35px;
+
+@bl-customComponent-rating-disabled: 0.38;
+@bl-customComponent-rating-labelWidth: 100%;
+
+.bl-customComponent-rating {
+  display: flex;
+  position: relative;
+
+  &.disabled {
+    opacity: @bl-customComponent-rating-disabled;
+    pointer-events: none;
+    cursor: default;
+  }
+}
+
+.bl-customComponent-rating__icons {
+  display: inline-flex;
+}
+
+.rating-icon {
```

Comment:
all classes without customComponent part should be inside *-customComponent-* class

```
.foo-customComponent-bar {
 .any{}
}
```